.changes/v3.9.0/965-features.md

```diff
@@ -0,0 +1,2 @@
+* **New Resource:** `vcd_rde_interface` to manage the Defined Interfaces required to use RDE types [GH-965]
```

Comment:
I had two options here:

- Name it with a more "canonical" `vcd_defined_interface`
- Name it with a grouping `rde` prefix: `vcd_rde_interface`
- A mix of both: `vcd_rde_defined_interface`

I chose the second option but I have my doubts as it could be confusing.